app/rtkrcv/rtk_skylark_l1l5.conf

```diff
@@ -0,0 +1,151 @@
+out-solstatic      =all        # (0:all,1:single)
+out-nmeaintv1      =0          # (s)
+out-nmeaintv2      =0          # (s)
+out-outstat        =off        # (0:off,1:state,2:residual)
+stats-eratio1      =100
+stats-eratio2      =100
+stats-errphase     =0.01       # (m)
+stats-errphaseel   =0.01       # (m)
+stats-errphasebl   =0          # (m/10km)
+stats-errdoppler   =1          # (Hz)
+stats-stdbias      =30         # (m)
+stats-stdiono      =0.03       # (m)
+stats-stdtrop      =0.3        # (m)
+stats-prnaccelh    =10         # (m/s^2)
+stats-prnaccelv    =10         # (m/s^2)
+stats-prnbias      =0.0001     # (m)
+stats-prniono      =0.001      # (m)
+stats-prntrop      =0.0001     # (m)
+stats-prnpos       =0          # (m)
+stats-clkstab      =5e-12      # (s/s)
+ant1-postype       =llh        # (0:llh,1:xyz,2:single,3:posfile,4:rinexhead,5:rtcm,6:raw)
+ant1-pos1          =90         # (deg|m)
+ant1-pos2          =0          # (deg|m)
+ant1-pos3          =-6335367.6285 # (m|m)
+ant1-anttype       =
+ant1-antdele       =0          # (m)
+ant1-antdeln       =0          # (m)
+ant1-antdelu       =0          # (m)
+ant2-postype       =rtcm       # (0:llh,1:xyz,2:single,3:posfile,4:rinexhead,5:rtcm,6:raw)
+ant2-pos1          =0          # (deg|m)
+ant2-pos2          =0          # (deg|m)
+ant2-pos3          =0          # (m|m)
+ant2-anttype       =
+ant2-antdele       =0          # (m)
+ant2-antdeln       =0          # (m)
+ant2-antdelu       =0          # (m)
+ant2-maxaveep      =3600
+ant2-initrst       =on         # (0:off,1:on)
+misc-timeinterp    =off        # (0:off,1:on)
+misc-sbasatsel     =0          # (0:all)
+misc-rnxopt1       =
+misc-rnxopt2       =
+misc-pppopt        =
+file-satantfile    =
+file-rcvantfile    =
+file-staposfile    =
+file-geoidfile     =
+file-ionofile      =
+file-dcbfile       =
+file-eopfile       =
+file-blqfile       =
+file-tempdir       =
+file-geexefile     =
+file-solstatfile   =
+file-tracefile     =
+
+inpstr1-type       =ntripcli   # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr2-type       =ntripcli   # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr3-type       =off        # (0:off,1:serial,2:file,3:tcpsvr,4:tcpcli,7:ntripcli,8:ftp,9:http)
+inpstr1-path       =username:password@igs-ip.net:2101/STFU00USA0
+inpstr2-path       =username:password@na2.skylark.swiftnav.com:2101/CRS
+inpstr3-path       =
+inpstr1-format     =rtcm3      # (0:rtcm2,1:rtcm3,2:oem4,3:oem3,4:ubx,5:ss2,6:hemis,7:skytraq,8:gw10,9:javad,10:nvs,11:binex,12:rt17,13:sbf,14:cmr,17:sp3)
+inpstr2-format     =rtcm3      # (0:rtcm2,1:rtcm3,2:oem4,3:oem3,4:ubx,5:ss2,6:hemis,7:skytraq,8:gw10,9:javad,10:nvs,11:binex,12:rt17,13:sbf,14:cmr,17:sp3)
+inpstr3-format     =rtcm3      # (0:rtcm2,1:rtcm3,2:oem4,3:oem3,4:ubx,5:ss2,6:hemis,7:skytraq,8:gw10,9:javad,10:nvs,11:binex,12:rt17,13:sbf,14:cmr,17:sp3)
+inpstr2-nmeareq    =latlon     # (0:off,1:latlon,2:single)
```

Comment:
Suggest changing this to `single` then remove `inpstr2-nmealat` and `inpstr2-nmealon`

Reply:
Actually during my testing, when setting to `single` mode, there was no skylark observations coming in. Only when specifying the lat and lon, it worked.

Reply:
Hmmm.... I will try and reproduce the problem on the bench.  It'll be a problem for customers if they need to use a hard-coded lat/lon!

Reply:
Figured out the reason. There are some IGS or GA's stations only sending observation messages, in such a case, SPP doesn't have a solution, and then corrections can not be sent. Changed it to `single`, but should be noted that if no ephemeris provided on the user end, the user needs to specify the lat and lon.